src/main/java/com/google/sps/data/EntityConstants.java

```diff
@@ -22,5 +24,6 @@
     public final static String SESSION_ID = "SessionId";
     public final static String SCREEN_NAME_OF_CONTROLLER = "ScreenNameOfController";
     public final static String IP_OF_VM = "IpOfVM";
+    public final static String LIST_OF_ATTENDEES = "ListOfAttendee";
```

Comment:
You dont need to store this. If you want to know all of the attendees of a session (on the server-side), you can write a query that grabs attendees by session id. On the server side, we dont need the session class or table to have a list of attendees.